packages/typespec-autorest/src/openapi.ts

```diff
@@ -1690,9 +1690,9 @@ export async function getOpenAPIForService(
 
   function resolveProperty(prop: ModelProperty, context: SchemaContext): OpenAPI2SchemaProperty {
     let propSchema;
-    if (prop.type.kind === "Enum" && prop.default) {
+    if (prop.type.kind === "Enum" && prop.defaultValue) {
```

Comment:
hmm, why all the value type fixes here?

Reply:
Technically not related to this PR, just I think the eslint pipeline not reporting the deprecated in the ci anymore, I need to investigate. YOu just see them in the IDE as warning.